Controllers/PatientsController.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using WebClinic.Data;
+using WebClinic.Models;
+
+// For more information on enabling MVC for empty projects, visit https://go.microsoft.com/fwlink/?LinkID=397860
+
+namespace WebClinic.Controllers
+{
+    [ApiController]
+    [Route("[controller]/[action]")]
+    public class PatientsController : Controller

```

Comment:
Why can't we use the existing Api/PatientsController?

Reply:
Why split them into API and non-api directories?

Reply:
So we are not supposed to have api/gui split?

Reply:
don't know, we'll see today

Reply:
Splitting/merging gui and api will be a less nightmare than rebasing my commits with yours new features :P

Reply:
well, that's the beauty of git :P